__tests__/app.test.js

```diff
@@ -123,7 +123,99 @@ describe("/api/reviews/:review_id", () => {
   });
 
   describe("patch request", () => {
-    test("should ", () => {});
+    test("should update votes of the review ", () => {
+      const updateInfo = {
+        inc_votes: 1
+      }
+
+      return request(app)
+        .patch("/api/reviews/1")
+        .send(updateInfo)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.review).toMatchObject({
+              title: 'Agricola',
+              designer: 'Uwe Rosenberg',
+              owner: 'mallionaire',
+              review_img_url:
+                'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+              review_body: 'Farmyard fun!',
+              category: 'euro game',
+              created_at: expect.any(String),
+              votes: 2
+          })
+        });
+    });
+
+    test("should give error if the body input does not match the object format ", () => {
+      const updateInfo = {
+        votes: 1
+      }
+
+      return request(app)
+        .patch("/api/reviews/1")
+        .send(updateInfo)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Input data format was not correct")
+        });
+    });
+
+    test('give error if invalid id is given', () => {
+      const updateInfo = {
+        inc_votes: 1
+      }
+
+      return request(app)
+        .patch("/api/reviews/100")
+        .send(updateInfo)
+        .expect(404)
```

Comment:
the description to this test implies that 100 is invalid, but this is in fact a valid ID it simply does not exist.